cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/xcm_config.rs

```diff
@@ -261,3 +279,22 @@ impl cumulus_pallet_xcm::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 }
+
+pub struct XcmFeeManagerFromComponentsBridgeHub<WaivedLocations, HandleFee>(
+	PhantomData<(WaivedLocations, HandleFee)>,
+);
+impl<WaivedLocations: Contains<Location>, FeeHandler: HandleFee> FeeManager
+	for XcmFeeManagerFromComponentsBridgeHub<WaivedLocations, FeeHandler>
+{
+	fn is_waived(origin: Option<&Location>, fee_reason: FeeReason) -> bool {
+		let Some(loc) = origin else { return false };
+		if let Export { network, destination: Here } = fee_reason {
+			return !(network == EthereumNetwork::get())
```

Comment:
Don't short-circuit when export network is not Ethereum.

```suggestion
			if network == EthereumNetwork::get() {
				return false
			}
```

Reply:
https://github.com/paritytech/polkadot-sdk/pull/5074/commits/909c40cfdfad2f552fa14a53bb83e1a2384d2f73